README.md

```diff
@@ -1,5 +1,24 @@
-# Packaging Library for use with Salesforce Packaging Platform
+# Salesforce Packaging
 
 > :warning: **This module is under heavy development, please do not use in production.**
 
-[![NPM](https://img.shields.io/npm/v/@salesforce/packaging.svg?label=@salesforce/packaging)](https://www.npmjs.com/package/@salesforce/packaging) [![CircleCI](https://circleci.com/gh/forcedotcom/packaging/tree/main.svg?style=shield)](https://circleci.com/gh/forcedotcom/packaging/tree/main) [![Downloads/week](https://img.shields.io/npm/dw/@salesforce/packaging.svg)](https://npmjs.org/package/@salesforce/packaging) [![License](https://img.shields.io/badge/License-BSD%203--Clause-brightgreen.svg)](https://raw.githubusercontent.com/forcedotcom/packaging/main/LICENSE.txt)
+[![NPM](https://img.shields.io/npm/v/@salesforce/packaging.svg?label=@salesforce/packaging)](https://www.npmjs.com/package/@salesforce/packaging) [![Downloads/week](https://img.shields.io/npm/dw/@salesforce/packaging.svg)](https://npmjs.org/package/@salesforce/packaging) [![License](https://img.shields.io/badge/License-BSD%203--Clause-brightgreen.svg)](https://raw.githubusercontent.com/forcedotcom/packaging/main/LICENSE.txt)
+
+## Description
+
+A JavaScript library for packaging metadata in your Salesforce project. This library supports both First-Generation packaging (1GP) and Second-Generation packaging (2GP).
```

Comment:
TypeScript?

Reply:
Both are true.  :)
I'll change it to TS though.

Reply:
I don't want to work out of the `lib/` dir though 😉